src/current/v24.1/plpgsql.md

```diff
@@ -409,6 +459,114 @@ BEGIN
   END
 ~~~
 
+### Perform transaction control
+
+Use a `COMMIT` or `ROLLBACK` statement within a PL/pgSQL [stored procedure]({% link {{ page.version.version }}/stored-procedures.md %}) to finish the current transaction and start a new one. 
```

Comment:
"and start a new one"? I thought you would have to issue a new `BEGIN`? Ahh I see this on line 483 -- perhaps something can be moved around?

Reply:
Hmm, just adding "automatically" here. The issue is that `BEGIN` does something totally different in PL/pgSQL.